### ISSUE TITLE
Word and phrase list has grown too long

### ISSUE THREAD

**Issue by foolip** [web-platform-dx/web-features#1527]
https://github.com/web-platform-dx/web-features/tree/main/docs#word-and-phrase-list is increasingly hard to follow for a new contributor, and to enforce as a reviewer, as the list grows.

I think that ideally the guidelines list some good examples of common cases that can be copy-pasted as a starting point.

What can simplify away and leave to undocumented convention?

**Comment by ddbeck** [comment]
> some good examples of common cases that can be copy-pasted as a starting point

So the very first item in https://github.com/web-platform-dx/web-features/tree/main/docs#descriptions was meant to be that, though nobody else has contributed to it. I could imagine a bit of an expansion, but I'm reluctant to do it because I'm not sure anyone has actually used it. Instead, I see a lot of cribbing from specifications (or MDN, which is often the same thing)—and I'm not quite sure how to discourage that behavior.

> What can simplify away and leave to undocumented convention?

I don't like this idea, honestly. I _don't_ expect folks to follow every item in the word list (I am absolutely certain that no tech writer at Google has the [Google developer documentation style guide word list](https://developers.google.com/style/word-list) memorized). Rather, I think it's a reference for repeated repetitions. If I read a description, I want to be able to distinguish between "I don't like this, but it's just my opinion" versus "we've had a discussion about it and here's what we settled on." If we move almost everything into the former category, then reviewers will lose the expectation to consider what other reviewers would or should do. I think you'd find my reviews _more_ insufferable without the style guide, is what I'm saying. 😄 

One thing that might help a lot though, to shorten the review loop, would be to adopt a tool like [Vale](https://vale.sh/) to check the strings automatically. There are a number of entries in the word list that I feel pretty confident could be automatically checked before a PR even opens.

**Comment by foolip** [comment]
Since I was involved in most of the discussions leading to a guideline update, I have the current word and phrase list mostly memorized, and a little bell goes ding when I see these words in review. But eventually the list would be too long for this to work, and I think it'll be a problem when we onboard more contributors and reviewers.

If most contributors start from draft features, perhaps the most important thing we could do here is to replace `description: TODO` in drafts with `description: TODO, see https://github.com/web-platform-dx/web-features/tree/main/docs#descriptions` so that this is the first thing people read. (I think adding examples for the very common cases of APIs, CSS properties, and HTML elements, would be a useful addition.)

To shorten the review time, it would good if reviewers update descriptions and then merge without another roundtrip. I currently don't do this, wanting to keep the PR author in the loop, but I think it'd be better overall. It's easy to send follow-up PRs, after all.

For the word and phrase list, it might be helpful to distinguish between recommended patterns like "also known as" and words to avoid. I've never used Vale, but if we could turn all "avoid" words into a lint, I think that would be an improvement. Going through the exercise of vetting existing usage of these words might also be helpful.
